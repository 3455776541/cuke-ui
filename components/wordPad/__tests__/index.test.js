import React from "react";
import { render } from "enzyme";
import toJson from "enzyme-to-json";
import WordPad from "../index";

describe("<WordPad/>", () => {
  it("should render a <WordPad/> components", () => {
    const wrapper = render(<WordPad />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
