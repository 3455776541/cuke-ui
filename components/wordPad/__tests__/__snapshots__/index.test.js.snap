// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WordPad/> should render a <WordPad/> components 1`] = `
<canvas
  class="cuke-word-pad"
>
  你的浏览器不支持 canvas
</canvas>
`;
